etc/config/systemd/ting_api.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Ting API Service
+Documentation=https://github.com/dionyziz/ting
+After=network.target
+
+[Service]
+WorkingDirectory=/var/www/ting.gr/html/API
+ExecStart=/var/www/ting.gr/html/API/venv/bin/python manage.py runserver
```

Comment:
These directories are not standard, meaning someone will probably have to go change them in order to utilize this service file.